itest/lnd_channel_balance_test.go

```diff
@@ -63,10 +63,6 @@ func testChannelBalance(ht *lntest.HarnessTest) {
 
 	// Ensure Bob currently has no available balance within the channel.
 	checkChannelBalance(bob, 0, amount-lntest.CalcStaticFee(cType, 0))
```

Comment:
How are channels and nodes now closed/shutdown to get a fresh state for the next test? Maybe I missed that in the review so far.

Reply:
So from now on all the nodes are created in each test independently, and are destroyed when the test ends, which means no more standby nodes that will carry their states into other tests.